### PR TITLE
CORE-1161 Add some error codes and get rid of client_id field

### PR DESCRIPTION
**Pull request by viccomx**
Signed-off-by: Victor Cruz <victorcruz@Bitsos-MacBook-Pro.local>

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1824,7 +1827,7 @@ Field Name | Type | Description | Units
 **side** | String | The order side (buy, sell) | -
 **status** | String | The order's status (queued, open, partial-fill, closed) | -
 **type** | String | The order type (market, limit) | -
-**client_id** | String | The client_id if any | -
+**origin_id** | String | The origin_id if any | -
```
Comment (CrispinFlowerday) [inline source/index.html.md]: What is the plan to change the API endpoints that have `client_id` in their name?

Comment (viccomx) [reply source/index.html.md]: Will create the tasks to change them, and follow the query param for doing the same.

Comment (CrispinFlowerday) [reply source/index.html.md]: Shouldn't you remove them from the docs now if you want to change them? And Notify any users?

Comment (viccomx) [reply source/index.html.md]: They should be removed from the documentation but we can keep them for those users that are already using it, a metric can be added to identify which users keep using them and then we can contact them so w can remove the method at all.

File: source/index.html.md [hunk 2]
```diff
@@ -182,6 +182,9 @@ error categories, the last two digits define specific errors.
 ### Unsupported HTTP method (400 error)
 * 0901: Unsupported HTTP method
 
+### Withdrawals error codes
+* 1000: "Error processing the withdrawal"
```
Comment (viccomx) [inline source/index.html.md]: This error code is now used See. https://github.com/bitsoex/slate/pull/45

Please change it

Comment (viccomx) [reply source/index.html.md]: Fixed

**Pull request merged**